### PR TITLE
LPS-122722 Add maximum width to translation row

### PR DESCRIPTION
**Pull request by brunofariasdeo**
LPS: https://issues.liferay.com/browse/LPS-122772

Hi @liferay-frontend ,

On this pull request I have turned the div that contains the plus button from child to sibling of languages selected on this tag lib. This was required to make sure that both would be on the same line and I could not find any workaround that would include changing only css.
I have also changed the width on Forms css side, to ensure that it would have a wrap effect once the languages added could not fit in the page width.
As per my investigation, this taglib is only used by Forms.

Kindly request your review and let me know if any further changes are required.

Test steps:

1. Go to Content and Data > Forms;
2. Click on the plus icon in order to create a new Form;
3. Locate the language submenu on top of the form page and try to add as many languages as possible.

Current state:

translation_manager_current

Expected:

image

### CODE REVIEW

**Comment by brunofariasdeo** [comment]
ci:test:sf

**Comment by brunofariasdeo** [comment]
ci:test:relevant

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/src/main/resources/META-INF/resources/admin/css/_portlet-forms-admin.scss [hunk 1]
```diff
@@ -1,6 +1,9 @@
 @media (min-width: 576px) {
 	.ddm-form-builder,
 	.ddm-translation-manager {
+		.autofit-row > .autofit-col:first-child {
+			width: 100%;
+		}
 		.modal-sm {
```
Comment (kresimir-coko) [inline modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/src/main/resources/META-INF/resources/admin/css/_portlet-forms-admin.scss:7]: I'd expect an empty line between these. All other selectors in the file are separated by an empty line.

File: modules/apps/frontend-taglib/frontend-taglib/src/main/resources/META-INF/resources/translation_manager/index.js [hunk 2]
```diff
@@ -118,21 +118,23 @@ const TranslationManager = ({
 								setVisibleModal(true);
 							}}
 						/>
+						<div className="autofit-col">
```
Comment (kresimir-coko) [inline modules/apps/frontend-taglib/frontend-taglib/src/main/resources/META-INF/resources/translation_manager/index.js:121]: Same as my comment above, I believe there should be an empty line between these two elements. Not sure why SF didn't pick it up.

**Comment by kresimir-coko** [comment]
Hey @brunofariasdeo I've tested your changes and apart from the 2 minor inline comments I approve these changes, here's how it looks for me with your changes 👍 

image

**Review by kresimir-coko — APPROVED** [review]

**Comment by carloslancha** [comment]
Hey @brunofariasdeo could you please add a test plan in the description (steps to follow to reproduce the issue)? Before and after screenshots would help us too.

We're trying to follow [some guidelines](https://github.com/liferay/liferay-frontend-guidelines/blob/master/general/pull_requests.md#prefer) in the pr so everyone could review it easily.

Thx!!

**Comment by brunofariasdeo** [comment]
> Hey @brunofariasdeo could you please add a test plan in the description (steps to follow to reproduce the issue)? Before and after screenshots would help us too.
> 
> We're trying to follow [some guidelines](https://github.com/liferay/liferay-frontend-guidelines/blob/master/general/pull_requests.md#prefer) in the pr so everyone could review it easily.
> 
> Thx!!

Hello @carloslancha,

Please find below the steps to reproduce the issue:

1. Go to Content and Data > Forms;
2. Click on the plus icon in order to create a new Form;
3. Locate the language submenu on top of the form page and try to add as many languages as possible.

Current state:

translation_manager_current

Expected:

image

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/src/main/resources/META-INF/resources/admin/css/_portlet-forms-admin.scss [hunk 3]
```diff
@@ -1,6 +1,9 @@
 @media (min-width: 576px) {
 	.ddm-form-builder,
 	.ddm-translation-manager {
+		.autofit-row > .autofit-col:first-child {
+			width: 100%;
```
Comment (jbalsas) [inline modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/src/main/resources/META-INF/resources/admin/css/_portlet-forms-admin.scss:5]: @marcoscv-work , @eduardoallegrini, @pat270, can you take a look at this? Without changing a lot more... is this the best approach we have?

Comment (eduardoallegrini) [reply modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/src/main/resources/META-INF/resources/admin/css/_portlet-forms-admin.scss:5]: I think we could use `.autofit-col-expand` @brunofariasdeo can you try with your changes?
ezgif com-gif-maker

**Review by eduardoallegrini — CHANGES_REQUESTED** [review]

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/src/main/resources/META-INF/resources/admin/css/_portlet-forms-admin.scss (hunk 3 above)
Comment (pat270) [reply modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/src/main/resources/META-INF/resources/admin/css/_portlet-forms-admin.scss:5]: :+1: for @eduardoallegrini suggestion

**Comment by brunofariasdeo** [comment]
Hi @eduardoallegrini ,
Thanks for your suggestion, using `.autofit-col-expand` was a great solution.

This is how it looks with my previous changes:

translation_manager_autofit-col-expand

**Review by eduardoallegrini — APPROVED** [review]
LGTM!

**Comment by jbalsas** [comment]
ci:forward

**Pull request closed without merging**